packages/relay-kit/src/packs/safe-4337/types.ts

```diff
@@ -144,6 +146,8 @@ export type EstimateFeeFunctionProps = {
   bundlerUrl: string
   entryPoint: string
   paymasterOptions?: PaymasterOptions
+  nodeUrl?: string
+  chainId?: bigint
```

Comment:
You can use `eth_chainId` instead passing it around

Reply:
the chain id is a fixed value, calling eth_chainId will add unnecessary latency.